include/boost/geometry/algorithms/append.hpp

```diff
@@ -91,21 +91,22 @@ template <typename Polygon, typename Point>
 struct point_to_polygon
 {
     typedef typename ring_type<Polygon>::type ring_type;
+	typedef typename interior_return_type<Polygon>::type interior_ring_range_type;
 
     static inline void apply(Polygon& polygon, Point const& point,
                 int ring_index, int = 0)
     {
         if (ring_index == -1)
         {
-			auto&& ext_ring = exterior_ring(polygon);
+			ring_type& ext_ring = exterior_ring(polygon);
             append_point<ring_type, Point>::apply(
-                        ext_ring, point);
+				ext_ring, point);
```

Comment:
@xventura81 Has your editor changed spaces vs tabs? Not sure what happened here.

Reply:
Yeah, Visual Studio does it by default.

Reply:
IMHO, that is not good to let accidental style changes happen. Every editor can be controlled to prevent it from replacing spaces with tabs or auto-reformatting. Imagine N authors working on common code base use N editors and each editor has different auto-formatting style as default... :-)

Reply:
You are right. Let's revert those changes in the indentation.